pkg/deps.bzl

```diff
@@ -14,40 +14,37 @@
 
 """WORKSPACE dependencies for rules_pkg/pkg."""
 
-load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", _http_archive = "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 
-# @federation: BEGIN @rules_pkg
+def http_archive(**kwargs):
```

Comment:
Really? I hate redefinitions of common things.
Of course, if I want to complain, http_archive should have had a "maybe" option from the beginning.